### PR TITLE
chore(client): add support for realtime user monitoring

### PR DESCRIPTION
**Pull request by overbit**
Close: https://github.com/amplication/amplication-cluster-configuration/issues/190

## PR Details

Add client monitoring leveraging RUM (Realtime-User-Monitoring). 
image

## PR Checklist

- [ ] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

**Comment by yuval-hazaz** [comment]
@overbit I like this change
We need to make sure the client works normally without the keys and connection to the RUM account.

**Comment by overbit** [comment]
@yuval-hazaz  @mulygottlieb we could have perhaps a quick chat about the next actions?

File: packages/amplication-client/src/util/monitoring.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    const APPLICATION_REGION = "us-east-1";
+
+    monitoring = new AwsRum(
+      REACT_APP_RUM_APP_ID,
+      APPLICATION_VERSION,
+      APPLICATION_REGION,
+      config
+    );
+  } catch (error) {
+    // Ignore errors thrown during CloudWatch RUM web client initialization
```
Comment (abrl91) [inline packages/amplication-client/src/util/monitoring.ts:38]: I think we should add `console.error(error);`

Comment (overbit) [reply packages/amplication-client/src/util/monitoring.ts:38]: No, I don't think we need to do so as it won't give us any thing we would act on.

**Review by abrl91 — APPROVED** [review]

**Comment by overbit** [comment]
Reopened as we had to revert due to issues with tracing

**Pull request merged**